scripts/actions/send-and-update-translation-queue.js

```diff
@@ -158,27 +158,22 @@ const uploadFiles = async (batches, translationsPerLocale, accessToken) => {
   for (const batch of batches) {
     let successCount = 0;
 
-    await Promise.all(
-      translationsPerLocale[batch.locale].map(async (translation) => {
-        const fileUploadResponse = await uploadFile(
-          batch.locale,
-          batch.batchUid,
-          accessToken
-        )(translation);
-
-        if (fileUploadResponse.code === 'ACCEPTED') {
-          await Database.updateTranslation(translation.id, {
-            status: 'IN_PROGRESS',
-          });
-          await Database.addTranslationsJobsRecord(translation.id, batch.jobId);
-          successCount += 1;
-        } else {
-          await Database.updateTranslation(translation.id, {
-            status: 'ERRORED',
-          });
-        }
-      })
-    );
+    const translations = translationsPerLocale[batch.locale];
+    for (const translation of translations) {
+      const fileUploadResponse = await uploadFile(
+        batch.locale,
+        batch.batchUid,
+        accessToken
+      )(translation);
+
+      if (fileUploadResponse.code === 'ACCEPTED') {
+        await Database.updateTranslation(translation.id, {
+          status: 'IN_PROGRESS',
+        });
+        await Database.addTranslationsJobsRecord(translation.id, batch.jobId);
+        successCount += 1;
+      }
```

Comment:
we probably want to wrap the body in a try catch so that each translation gets a chance to be uploaded, even if a previous translation encounters an error.

Reply:
it looks like `uploadFile` will handle everything except for Fetch throwing on a network error. A Try/Catch here sounds perfect!